Source/WebCore/html/ImageBitmap.cpp

```diff
@@ -732,6 +732,12 @@ class PendingImageBitmap final : public ActiveDOMObject, public FileReaderLoader
         m_createImageBitmapTimer.suspendIfNeeded();
     }
 
+    ~PendingImageBitmap()
+    {
+        if (m_completionHandler)
+            std::exchange(m_completionHandler, std::nullopt).value()(Exception { InvalidStateError, "PendingImageBitmap is being destroyed"_s });
```

Comment:
You don't need to std::exchange, calling the completion handler will nullify it.